src/Assert.php

```diff
@@ -719,6 +721,16 @@ public static function notRegex($value, $pattern, $message = '')
         }
     }
 
+    public static function letter($value, $message = '')
+    {
+        if (!preg_match('/^\p{L}+$/u', $value)) {
```

Comment:
To avoid php warning/ unexpected behaviour, it may be better to add a `static::string($value);` before, to make sure that the passed value is in fact a string.

Reply:
I just copied the style from `alpha()` which doesn't check for a string. I'll add this check.

Reply:
Alpha uses ctype which silently fails for any non string input. Regex may complain if provided with an array for example

Reply:
Good point!